packages/i18next/src/lib/I18nextHandler.ts

```diff
@@ -25,11 +31,23 @@ export class I18nextHandler {
 
 	/**
 	 * The options I18nextHandler was initialized with in the client.
+	 * @since 1.0.0
 	 */
 	public readonly options?: I18nOptions;
 
-	private readonly languagesDir: string;
-	private readonly backendOptions: i18nextFsBackend.i18nextFsBackendOptions;
+	/**
+	 * The director passed to `i18next-fs-backend`.
+	 * Also used in {@link I18nextHandler#walkLanguageDirectory}.
```

Comment:
```suggestion
	 * Also used in {@link I18nextHandler.walkLanguageDirectory}.
```
Typedoc will not use `Object#methodName` with linking. This suggestion should be changed to link correctly in docs.